<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>appNano</class>
 <widget class="QDialog" name="appNano">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Aplicacio Arduino NANO</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
background-color: beige;
border: 2px solid green;
}</string>
  </property>
  <widget class="QLabel" name="etqPort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
border: 1px solid blue;
}</string>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="botoConectar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>5</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Conecta la Placa</string>
   </property>
  </widget>
  <widget class="QLabel" name="etqOnOff">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>8</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../imatges/vermell.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="etqEstat">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Estat: Desconectat</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gr_0">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Pin 0 Rx0</string>
   </property>
   <widget class="QLabel" name="IO_0">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imatges/vermell.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="boto_0">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Activar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="desa_0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>I O</string>
    </property>
   </widget>
   <widget class="QLabel" name="etq_0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gr_1">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Pin 1 Tx1</string>
   </property>
   <widget class="QLabel" name="IO_1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imatges/vermell.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="boto_1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Activar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="desa_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>I O</string>
    </property>
   </widget>
   <widget class="QLabel" name="etq_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gr_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>141</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Pin 2 D2</string>
   </property>
   <widget class="QLabel" name="IO_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imatges/vermell.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="boto_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Activar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="desa_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>I O</string>
    </property>
   </widget>
   <widget class="QLabel" name="etq_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gr_3">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>141</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox{
background-color: aqua;
}</string>
   </property>
   <property name="title">
    <string>Pin 3 D3</string>
   </property>
   <widget class="QLabel" name="IO_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imatges/vermell.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="S_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>71</width>
      <height>24</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
   </widget>
   <widget class="QPushButton" name="resta_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imatges/resta.png</normaloff>../imatges/resta.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="suma_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imatges/suma.png</normaloff>../imatges/suma.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="etqD_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>40</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="d_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="boto_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>80</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Activar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="desa_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>IO SP</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
